great_expectations/datasource/fluent/pandas_dbfs_datasource.pyi

```diff
@@ -53,6 +54,7 @@ class PandasDBFSDatasource(PandasFilesystemDatasource):
         batching_regex: Optional[Union[str, re.Pattern]] = ...,
         glob_directive: str = ...,
         order_by: Optional[SortersDefinition] = ...,
+        batch_metadata: Optional[BatchMetadata] = ...,
```

Comment:
Is `None` ever an acceptable value here?
Not a question specifically for this method but all of `batch_metadata: Optional[BatchMetadata] = ...,`